### PR TITLE
New WL code into CAVA

### PR DESCRIPTION
**Pull request by claalmve**
# Description of PR

### Summary of changes and related issue
Integrating new WL back-end from #449 into CAVA.

### Relevant motivation and context
This is so we can use more recent WL code in the CAVA and so that it doesn't break later.

### Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

<br>

#### Practical
- [ ] 80% unit test coverage
- [x] Documentation
  - [ ] All functions/adjusted functions documented in the [readthedocs](https://climakitae.readthedocs.io/en/latest/).
  - [ ] Documentation is pushed
- [x] Complex code commented
- [x] Naming conventions followed
  - [ ] Helper functions hidden with `_` before the name
- [x] Context of function is clearly provided
  - [ ] Intent of function is provided
  - [ ] How to test, so that it is not siloed on scientists and anyone can review
  - [ ] Appropriate manual testing was completed
- [x] Any notebooks known to utilize the affected functions are still working
- [x] Linting completed and resolved

#### Conceptual
- [x] Doesn't replicate existing functionality
- [x] Aligns with general coding standard of existing functions
- [x] Matches desired functinonality from users/scientists

### CODE REVIEW

**Comment by claalmve** [comment]
This must be review in conjunction with https://github.com/cal-adapt/cae-notebooks/pull/135 .

**Comment by nicolejkeeney** [comment]
[WL integration: update CAVA notebooks/code to use new warming levels functionality with DataParameters](https://trello.com/c/rQ993D3S/751-wl-integration-update-cava-notebooks-code-to-use-new-warming-levels-functionality-with-dataparameters)

**Review by vicford — COMMENTED** [review]
Just a single comment, seemed to work fine in new nb

File: climakitae/explore/vulnerability.py [hunk 1]
```diff
@@ -81,6 +80,7 @@ def _filter_ba_models(data, downscaling_method, wrf_bias_adjust, historical_data
                         "WRF_MPI-ESM1-2-HR_r3i1p1f1",
                         "WRF_TaiESM1_r1i1p1f1",
                         "WRF_MIROC6_r1i1p1f1",
+                        "WRF_EC-Earth3-Veg_r1i1p1f1",
```
Comment (vicford) [inline climakitae/explore/vulnerability.py:83]: TY for adding!

File: climakitae/explore/vulnerability.py [hunk 2]
```diff
@@ -365,21 +365,21 @@ def validate_params(self):
...
-            "2.0",
-            "2.5",
-            "3.0",
-            "4.0",
+        if self.approach == "Warming Level" and self.warming_level not in [
+            1.5,
+            2.0,
+            2.5,
+            3.0,
+            4.0,
```
Comment (vicford) [inline climakitae/explore/vulnerability.py:380]: Is 4.0 an option now? If so, I'll amend the table again to include

File: climakitae/explore/vulnerability.py [hunk 3]
```diff
@@ -456,14 +456,14 @@ def get_names(self):
...
             approach_str = f"{self.time_start_year}_to_{self.time_end_year}"
             approach_var_name = f"from {approach_str}"
-        elif self.approach == "warming_level":
+        elif self.approach == "Warming Level":
             wl_str = (
-                str(int(float(self.warming_level)))
-                if self.warming_level not in ["1.5", "2.5"]
-                else self.warming_level.replace(".", "pt")
+                str(int(self.warming_level))
+                if self.warming_level not in [1.5, 1.5]
```
[the remark this answers is not in the document]
Comment (claalmve) [reply climakitae/explore/vulnerability.py:465]: Oh god yes thank you for seeing that

File: climakitae/explore/vulnerability.py (hunk 2 above)
Comment (claalmve) [reply climakitae/explore/vulnerability.py:380]: Let me remove that actually -- I'm not sure if that totally works with current CAVA design

**Pull request merged**